api/python/provisioner/commands/__init__.py

```diff
@@ -351,6 +352,138 @@ def run(self, *args, targets=ALL_MINIONS, **kwargs):
         return res
 
 
+@attr.s(auto_attribs=True)
+class SetNTP(CommandParserFillerMixin):
```

Comment:
`SetNTP` shouldn't copy-paste the whole `Set` class, please just inherit from the latter and redefine what is different